_drafts/newsletter-001.md

```diff
@@ -298,6 +406,15 @@ Special section for other news (up to 15) in a one-liner format:
   [a video of hot reloading demonstration](https://twitter.com/wodannson/status/1157472538622078976)
   \[[/r/rust](https://reddit.com/r/rust/comments/cldaew/hot_reloading_of_function_bodies_in_rust),
   [/r/rust_gamedev](https://reddit.com/r/rust_gamedev/comments/cldajt/hot_reloading_of_function_bodies_in_rust/)].
+- [rx](https://cloudhead.io/rx) - a minimalist and extensible pixel editor in Rust
+  \[[/r/rust](https://www.reddit.com/r/rust/comments/cv6o4q/announcing_rx_minimalist_and_extensible_pixel),
+  [repo](https://github.com/cloudhead/rx)].
+- [gfx-hal v0.3](https://github.com/gfx-rs/gfx/) -
+  new version of a high-performance, bindless graphics API, features:
```

Comment:
Let's change the description to "A low-overhead Vulkan-like GPU API for Rust". It's our fault that the wrong description was kept for so long... It stopped being "bindless" as soon as we moved to HAL.

Reply:
Done :)